gatsby/content/projects/clients/syphon.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: projectimage
+title: Syphon
+categories:
+ - client
+description: a not for profit, open source matrix client with a focus on privacy and ease of use
+author: Taylor Ereio
+home: https://syphon.org
+language: Dart/Flutter
+license: AGPL-3.0-or-later
+repo: https://github.com/syphon-org/syphon
+room: "#syphon:matrix.org"
+slug: syphon
+thumbnail: /docs/projects/images/syphon-thumb.png <TODO>
+screenshot: /docs/projects/images/syphon-full.png <TODO>
+maturity: Alpha
+platforms:
+    - Android
+    - iOS
+    - Linux
+    - Mac
+features:
+    Room directory: yes
+    Room tag showing: no
+    Room tag editing: no
+    Search joined rooms: no
+    Room user list: yes
+    Display Room Description: yes
+    Edit Room Description: yes
+    Highlights: no
+    Pushrules: no
+    Send read markers: yes
+    Display read markers: yes
+    Sending Invites: yes
+    Accepting Invites: yes
+    Typing Notification: yes
+    E2E: Partial
+    Replies: yes
+    Attachment uploading: no
+    Attachment downloading: no
+    Send stickers: no
+    Send formatted messages (markdown): no
+    Rich Text Editor for formatted messages: no
+    Display formatted messages: no
+    Redacting: no
+    Multiple Matrix Accounts: no
+    New user registration: yes
+    voip: no
+    Reactions: yes
+    Message editing: Partial
+    Room upgrades: no
+    Localisations: 2
+    SSO Support: no
```

Comment:
Syphon does support SSO, only one I've tested extensively however is mozilla's homeserver

Reply:
Ah, thanks for pointing that out. Fixed :D